### ISSUE TITLE
Junk QUIC High Hardware Usage

### ISSUE THREAD

**Issue by jobs-git** [quicwg/base-drafts#2795]
According to research by Google, hardware usage using QUIC can reach upto 3.5x that of TCP+TLS, when optimized, this can be reduced to 2x hardware usage that of TCP+TLS. 

This means that if QUIC/HTTP3 becomes a new standard that needs about 2x the hardware needed to operate the Internet in the whole world. In addition, this means that it will consume two times more energy than the current energy usage. 

In summary, QUIC is 2x costly and 2x more crippling to the World Wide Network.

Here is the research article

https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46403.pdf

**Comment by larseggert** [comment]
If true, this only remains true until we have better hardware offload functionality.

I see nothing actionable for the WG here.
